app/scraper/sources/face_book.py

```diff
@@ -137,9 +169,12 @@ def scrape(self, current_people):
                 'school_code': 'YC',
             }
 
-            person['last_name'], person['first_name'] = self.clean_name(container.find('h5', {'class': 'yalehead'}).text)
-            person['year'] = self.clean_year(container.find('div', {'class': 'student_year'}).text)
-            pronouns = container.find('div', {'class': 'student_info_pronoun'}).text
+            person['last_name'], person['first_name'] = self.clean_name(
+                container.find('h5', {'class': 'yalehead'}).text)
```

Comment:
I like to ignore the pep8 line length limit thing

Reply:
imo if we're gonna make this cleaner, we should make it expressively cleaner ourselves based on what the logic is actually doing rather than based on pycodestyle choosing a random place to break the line

Reply:
pep8 is standard convention so that's why my editor was set to that. workspace settings disabled format on save and should be good now.